client/test/io.js

```diff
@@ -0,0 +1,11 @@
+/**
+  * Mock class for io
+  */
+
+module.exports = () => ({
+  socket: {
+    postAsync: function () {
+      return Promise.resolve().then(() => ({}));
```

Comment:
Not sure if it matters, but this will return a native Promise whereas the actual `postAsync` function will return a bluebird Promise. This could cause confusing test failures in the future.